src/server/protocol.ts

```diff
@@ -1333,7 +1333,7 @@ namespace ts.server.protocol {
      * For external projects, some of the project settings are sent together with
      * compiler settings.
      */
-    export type ExternalProjectCompilerOptions = CompilerOptions & CompileOnSaveMixin & WatchOptions;
+    export type ExternalProjectCompilerOptions = CompilerOptions & CompileOnSaveMixin & WatchOptions & TypeAcquisition;

```

Comment:
`ExternalProject` has options called `typeAcquisition` so this type shouldnt be changed.

Reply:
You can create different type instead you should create a type 
`export type InferredProjectCompilerOptions = ExternalProjectCompilerOptions  & TypeAcquisition` and update the usage.